src/Maestro/SubscriptionActorService/DarcRemoteFactory.cs

```diff
@@ -20,18 +21,21 @@ public class DarcRemoteFactory : IRemoteFactory
             IConfigurationRoot configuration,
             IGitHubTokenProvider gitHubTokenProvider,
             IAzureDevOpsTokenProvider azureDevOpsTokenProvider,
+            IMemoryCache memoryCache,
```

Comment:
I don't see anywhere where this is actually set up other than in the website project. An IMemoryCache will need to be registered in the SubscriptionActoryService as well so it can be used by the thing actually creating PRs.

Reply:
I don't think it needs to be in the web service, right? There is no DarcRemoteFactory singleton added to that service.  I moved it over to the SubscriptionActorService.